hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -144,14 +143,8 @@ public class RocksDBCheckpointDiffer implements AutoCloseable,
   private static final String COMPACTION_LOG_ENTRY_INPUT_OUTPUT_FILES_DELIMITER
       = ":";
 
-  /**
-   * SST file extension. Must be lower case.
-   * Used to trim the file extension when writing compaction entries to the log
-   * to save space.
-   */
-  static final String SST_FILE_EXTENSION = ".sst";
   public static final int SST_FILE_EXTENSION_LENGTH =
-      SST_FILE_EXTENSION.length();
+      ManagedRocksDB.SST_FILE_EXTENSION.length();
```

Comment:
nit: static import `SST_FILE_EXTENSION`.

Reply:
We don't need it anymore